src/Core/Geometry/Adjacency.cpp

```diff
@@ -13,7 +13,7 @@ namespace Geometry {
 // ADJACENCY MATRIX //
 // //////////////// //
 
-AdjacencyMatrix uniformAdjacency( const uint point_size, const VectorArray<Vector3ui>& T ) {
+AdjacencyMatrix uniformAdjacency( const uint point_size, const AlignedStdVector<Vector3ui>& T ) {
```

Comment:
Well, it was at a time the indices could be something else than Eigen Vectors.
Since VectorArray can only contain EigenVectors, I modify this.
After some thinking, Eigen introspection is nice so I switch indices type to Eigen Vectors, so I may switch bach these parameter to VectorArray ... I will check.

Reply:
The only thing is that a VectorArray give memory mapping facilities.

Reply:
Why indices must be only eigen vectors. They are if they define individual triangles or primitives but for, e.g. triangle strips, and more generally primitive strips they are'nt. So alignedStdVector seems fine for me (and mapping facilities could be done also no ?)

Reply:
The current status of this discussion is not clear.
To we want to introduce this change in the PR, or leave it as it was before ?

Reply:
My point is :
Leave as is, if needed or clarified, we can switch back to AlignedStdVector